### ISSUE TITLE
Remove / rebuild the quality template aka editor toolkit

### ISSUE THREAD

**Issue by brendanheywood** [theCrag/website#934]
eg the old editors toolkit which is now busted is about half replaced by the facets pages.

http://www.thecrag.com/climbing/australia/queensland/cedar-creek/quality

But there was a lot it did that is now found nowhere. What I have in mind is a replacement simple page showing various stats and links off to various facet pages for more details.

Idea: Assign a 'technical grade' (ie automated tests below) vs 'adjectvial grade' (based on surveys from users who have climbed there)

eg it would have a bunch of stats like:

#### Names
- [ ] compare # of routes to estimate of # of routes at crag level
- [ ] show potentially duplicated routes
- [ ] show routes with 'closed' in their name but not tagged as closed
https://www.thecrag.com/en/climbing/australia/gara-gorge/upper-gara-gorge/routes/search/closed/?sortby=at,desc
- [ ] show routes with 'unknown', or other non-name things in them
https://www.thecrag.com/en/climbing/australia/gara-gorge/upper-gara-gorge/routes/search/unknown/?sortby=at,desc
- [ ] # of routes of unknown type
https://www.thecrag.com/en/climbing/australia/gara-gorge/upper-gara-gorge/routes/with-gear-style/unknown/?sortby=at,desc

#### Overall size
- [ ] if too many routes should be broken into sub cliffs (>100) (TODO new filter /areas/)
- [ ] if still large should have annotations (eg 5-10% annotations)
- [ ] check how evenly they are spread out
- [ ] every area should have at least 1 but ideally more children
- [ ] Adjectival: routes are missing

#### Fields and tags
- [ ] has various fields entered
- [ ] access recently updated
- [ ] # tags missing or show distribution of tags

#### History
- [ ] # of routes with no FA year or person (need to filter on EITHER missing year or names)
https://www.thecrag.com/en/climbing/australia/gara-gorge/upper-gara-gorge/routes/with-gear-style/trad+sport+top-rope+dws+aid+via-ferrata/assert/is-not-project+missing-history/?sortby=at,desc
- [ ] duplicate history with no explanatory note (this is legit to have two FA's or two FFA's)
- [ ] # tools to show the list of FA people to ensure consistency, and later link to real accounts if they exist
- [ ] if spotty history data in routes then must have a history cliff notes
- [ ] list of FA names has potential duplicates, search for similarities. Search for missing initials or initials instead of a first name (do at tlc level)
- [ ] search for FA's which match an account, especially an account which has karma in the same area. (do at tlc level)

#### Topos
- [ ] # topos without any linked OR drawn routes
- [ ] # routes which are drawn in more than 3 topos in the same cliff

#### Quality and stars
- [ ] show distribution of star rating, should be roughly a pareto / log normal distribution (ie look for no stars at all, and converserly look for areas where 'everything is classic'

#### Description / topo
- [ ] # of routes with no description AND no topo line (one or the other is fine, both is better)

#### Height / Length
- [ ] # of routes with no height AND not bouldering
https://www.thecrag.com/en/climbing/australia/gara-gorge/upper-gara-gorge/routes/with-gear-style/trad+sport+top-rope+dws+aid+via-ferrata/assert/missing-length/?sortby=at,desc

#### Sport
- [ ] # of sport routes with no bolts: 35 - and the 35 would be a link off to the facet page show those 35 routes.
https://www.thecrag.com/en/climbing/australia/gara-gorge/upper-gara-gorge/routes/with-gear-style/sport/assert/missing-bolts/?sortby=at,desc

#### Trad
- [ ] must have description

#### Grades
- [ ] # of routes with no grade or inconsistent grading system
- [ ] number of routes whose primary grade isn't in the normal context
- [ ] looks for 'foreign' grades which don't translate well to the primary grade
- [ ] show routes who have multiple alternate grades
- [ ] show some sort of grade distribution to help spot inconsistent grades

#### Geolocation
- [ ] missing polygon
- [ ] average locatedness each child node - see #844 
- [ ] rule of thumb, all routes should have locatedness around the same as it's height. ie boulders should have very specific locations, alpine are easier to spot at a distance
- [ ] areas polygons which overlap
- [ ] areas which are not inside their parent
- [ ] parents which shrink wrap union of children (ie can you ever be 'in' a crag but not in a cliff? eg car park)
- [ ] polygons which are twisted onto themselves
- [ ] areas with unusual polygon complexity, eg point count is large, or points are redundant
- [ ] polygons with very acute angles, eg < 20 degrees
- [ ] all leaf cliffs should have aspect or inherit
- [ ] if too many aspects should be broken up into sub cliffs

#### Multi-pitch
- [ ] look for very high routes which are not marked as multi pitch
- [ ] is multi pitch but no grades or height breakdown for each pitch
- [ ] is multi but no markdown pitch descriptions
- [ ] is multi sport but has ambiguous draw count (eg 50 draws total, vs 12 draws needed to climb it)
- [ ] pitch grade is higher than route grade
- [ ] topos should have belay points

It could do a nice simple color code for when things are bad or good as a nice quick 'health check'

Basically I'd like this page to serve the same purpose as the original toolkit: to help coordinators focus their efforts on gaps and improve the baseline.

It would be super awesome if this was somehow more transparent with the crag maturity stat.

It might also be good as another issue to randomly ask people for quality ratings on a crag when they tick a route to get a feel for the non easily measurable things like how well a description is written.

Things we can't measure directly, need to crowd source:
- [ ] are all descriptions spelling and grammar correct
- [ ] factual accuracy, match info to reality
- [ ] legibility / composition of topos

**Comment by killakalle** [comment]
+1
would love to have this as an editor.

Somewhat related to a discussion I started a while ago:
https://www.thecrag.com/discussion/3792870867/missing-crag-content-quality-and-completeness-indicator
